README.md

```diff
@@ -203,3 +203,432 @@ kubectl delete -k subscription/deployment-k8s-regional-rbd
+
+The sample application is configured to run on cluster `dr1`. To deploy
+it on cluster `dr1` and make it possible to fail over or relocate to
+cluster `dr2` we need to create the namespace on both clusters:
+
+```
+kubectl create ns deployment-rbd --context dr1
+kubectl create ns deployment-rbd --context dr2
+```
+
+To deploy the application apply the deployment-rbd workload to the
+`deployment-rbd` namespace on cluster `dr1`:
+
+```
+kubectl apply -k workloads/deployment/k8s-regional-rbd -n deployment-rbd --context dr1
+```
+
+To view the deployed application use:
+
+```
+kubectl get deploy,pod,pvc -n deployment-rbd --context dr1
+```
+
+Example output:
+
+```
+NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/busybox   1/1     1            1           24s
+
+NAME                           READY   STATUS    RESTARTS   AGE
+pod/busybox-6bbf88b9f8-fz2kn   1/1     Running   0          24s
+
+NAME                                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS      VOLUMEATTRIBUTESCLASS   AGE
+persistentvolumeclaim/busybox-pvc   Bound    pvc-c45a3892-167b-4dbc-a250-09c5f288c766   1Gi        RWO            rook-ceph-block   <unset>                 24s
+```
+
+## Enabling DR for OCM discovered application
+
+Unlike OCM managed applications, the DR resources for all applications
+are in the `ramen-ops` namespace.
+
+To prepare the `ramen-ops` namespaces apply the managed clusterset
+binding resource. This should be done once before enabling DR for
+discovered applications.
+
+```
+kubectl apply -f dr/discovered/ramen-ops/binding.yaml --context hub
+```
+
+Example output:
+
+```
+managedclustersetbinding.cluster.open-cluster-management.io/default created
+```
+
+To enable DR for the application, apply the DR resources to the hub
+cluster:
+
+```
+kubectl apply -k dr/discovered/deployment-rbd --context hub
+```
+
+Example output:
+
+```
+placement.cluster.open-cluster-management.io/deployment-rbd-placement created
+placementdecision.cluster.open-cluster-management.io/deployment-rbd-placement-decision created
+drplacementcontrol.ramendr.openshift.io/deployment-rbd-drpc created
+```
+
+To set the application placement, patch the placement decision resource
+in the `ramern-ops` namespace on the hub:
+
+```
+kubectl patch placementdecision deployment-rbd-placement-decision \
+    --subresource status \
+    --patch '{"status": {"decisions": [{"clusterName": "dr1", "reason": "dr1"}]}}' \
+    --type merge \
+    --namespace ramen-ops \
+    --context hub
+```
+
+Example output:
+
+```
+placementdecision.cluster.open-cluster-management.io/deployment-rbd-placement-decision patched (no change)
+```
+
+At this point *Ramen* take over and start protecting the application.
+
+*Ramen* creates a `VolumeReplicationGroup` resource in the `ramen-ops`
+namespace in cluster `dr1`:
+
+```
+kubectl get vrg -l app=deployment-rbd -n ramen-ops --context dr1
```

Comment:
This does not work:

```
$ kubectl get vrg -l app=deployment-rbd -n ramen-ops --context dr1
No resources found in ramen-ops namespace.
```

Because ramen does not label the vrg with the same label of the drpc. I think this works for OCM managed apps.